task/07-yield-tasks.js

```diff
@@ -33,10 +32,30 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+  let i = 99;
+  while (i > 0) {
+    if (i === 99) {
+      yield "99 bottles of beer on the wall, 99 bottles of beer.";
+    }
+    i--;
+    if (i > 1) {
+      yield "Take one down and pass it around, " +
+        i +
+        " bottles of beer on the wall.";
+      yield i + " bottles of beer on the wall, " + i + " bottles of beer.";
+    } else if (i === 1) {
+      yield "Take one down and pass it around, " +
+        i +
+        " bottle of beer on the wall.";
+      yield i + " bottle of beer on the wall, " + i + " bottle of beer.";
+    } else {
+      yield "Take one down and pass it around, no more bottles of beer on the wall.";
+      yield "No more bottles of beer on the wall, no more bottles of beer.";
+      yield "Go to the store and buy some more, 99 bottles of beer on the wall.";
+    }
+  }
 }
```

Comment:
this is a very not logical redundant solution
from 99 to 2 bottles it has the same template and after them just 4 lines more